TACHO_DRIVER.md

```diff
@@ -0,0 +1,233 @@
+Tacho Driver Update Proposal
```

Comment:
There's not much value in keeping this, I think. The test cases and actual code will decument what was actually done.